### PR TITLE
Dm 21132 - Initial commit

### PR DESCRIPTION
**Pull request by jbuffill**

### CODE REVIEW

File: schema/Hexapod.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    default: 1
+  v_max:
+    description: The max limit along the v-axis (rot about y-axis)
+    type: number
+    default: 1
+  w_max:
+    description: The max limit along the w-axis (rot about z-axis)
+    type: number
+    default: 1
+
```
Comment (tribeiro) [inline schema/Hexapod.yaml]: Can you remove the extra lines at the end? leave only one blank lines, no spaces..

File: setup.cfg [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504, E401, E303, E302, E305
```
Comment (tribeiro) [inline setup.cfg]: Please, remove the additional ignored flake8 tests... without them there's no guarantee the code will follow the project standards...

File: setup.cfg [hunk 3]
```diff
@@ -11,4 +11,4 @@ exclude =
 
 [tool:pytest]
 addopts = --flake8
-flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W504
+flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W504 E401 E303 E302 E305
```
Comment (tribeiro) [inline setup.cfg]: Please! Remove the additional ignored tests.

File: schema/Hexapod.yaml (hunk 1 above)
Comment (jbuffill) [reply schema/Hexapod.yaml]: Done

File: setup.cfg (hunk 2 above)
Comment (jbuffill) [reply setup.cfg]: The removed lint tests were too nit-picky, restrictive & provided no added value.

File: setup.cfg (hunk 3 above)
Comment (jbuffill) [reply setup.cfg]: The removed lint tests were too nit-picky, restrictive & provided no added value.

**Pull request closed without merging**